templates/part.board.mainView.php

```diff
@@ -138,6 +138,52 @@ class="icon icon-user"></span><span><?php p($l->t('Unassign from me')); ?></span
 											   ng-click="cardArchive(c); $event.stopPropagation();"><span
 														class="icon icon-archive"></span><span><?php p($l->t('Archive')); ?></span></a>
 										</li>
+										<li ng-if="params.filter!=='archive'">
+											<span class="menuitem permanent popover-form">
+											   <span class="icon icon-archive"></span>
+												<span ng-click="card.showMovePopOver = !card.showMovePopOver"><?php p($l->t('Move')); ?></span>
+												<div ng-show="card.showMovePopOver" class="popover-form-container">
+														<div class="form-row">
```

Comment:
Use ng-if here since otherwise the element is in the Dom for every card all the time ;)